code/game/machinery/teleporter.dm

```diff
@@ -254,7 +254,7 @@
 	if (istype(M, /mob/living))
 		var/mob/living/MM = M
 		if(MM.check_contents_for(/obj/item/weapon/storage/backpack/holding))
-			MM << "<span class='warning'>The Bluespace interface on your Bag of Holding interferes with the teleport!</span>"
+			Mto_chat(M, "<span class='warning'>The Bluespace interface on your Bag of Holding interferes with the teleport!</span>")
```

Comment:
issue

Reply:
Would probably be best just to delete the commentedout code.